leptos_dom/src/events.rs

```diff
@@ -150,7 +150,10 @@ pub(crate) fn add_delegated_event_listener(
                         if !maybe_handler.is_undefined() {
                             let f = maybe_handler
                                 .unchecked_ref::<js_sys::Function>();
-                            let _ = f.call1(&node, &ev);
+                            match f.call1(&node, &ev) {
```

Comment:
style: I don't know if `clippy` has a lint for this or not, but if you have an empty match branch I'd think
```rust
if let Err(e) = f.call1(&node, &ev) {
```
would be better